enterprise/coderd/provisionerdaemons.go

```diff
@@ -178,6 +178,14 @@ func (api *API) provisionerDaemonServe(rw http.ResponseWriter, r *http.Request)
 		}
 	}
 
+	var name string
+	if vals, ok := r.URL.Query()["name"]; !ok || len(vals) == 0 {
+		api.Logger.Warn(ctx, "unnamed provisioner daemon")
+		name = namesgenerator.GetRandomName(1)
```

Comment:
curious: 1 trial is enough?

Reply:
I guess I saw somewhere `namesgenerator.GetRandomName(10)`, but that could be an integration test.